src/main/java/org/elasticsearch/script/expression/DateMethodFunctionValues.java

```diff
@@ -25,13 +25,14 @@
 
 import org.apache.lucene.queries.function.ValueSource;
 import org.elasticsearch.index.fielddata.AtomicNumericFieldData;
+import org.elasticsearch.search.MultiValueMode;
 
 class DateMethodFunctionValues extends FieldDataFunctionValues {
     private final int calendarType;
     private final Calendar calendar;
 
-    DateMethodFunctionValues(ValueSource parent, AtomicNumericFieldData data, int calendarType) {
-        super(parent, data);
+    DateMethodFunctionValues(ValueSource parent, MultiValueMode multiValueMode, int calendarType, AtomicNumericFieldData data) {
```

Comment:
Can we keep the order of arguments for this and super ctor consistent? So all the args that will go to the super class should come first, in the same order to the super ctor? It is confusing to see args moved around (here the order is consistent but in others it seems not)